keyless/pepper/common/src/lib.rs

```diff
@@ -79,6 +79,42 @@ pub struct PepperResponse {
     pub signature: Vec<u8>, // unique BLS signature
 }
 
+/// A pepper scheme where:
+/// - The pepper input contains `JWT, epk, blinder, expiry_time, uid_key, aud_override`, wrapped in type `PepperRequestV1`.
+/// - `epk` variant is `EphemeralPublicKey::Ed25519`.
+/// - The pepper output is the `BLS12381_G1_BLS` VUF output of the input, encrypted by `epk`, wrapped in type `PepperResponseV1`.
+#[derive(Debug, Deserialize, Serialize)]
+pub struct PepperRequestV1 {
+    #[serde(rename = "jwt_b64")]
+    pub jwt: String,
+    #[serde(
```

Comment:
You can remove this; it's already implemented [here](https://github.com/aptos-labs/aptos-core/blob/a7a474f9c064dd7b05cda11e7fc0a7215f427403/types/src/transaction/authenticator.rs#L1149).

Reply:
We should unify this actually... Since we are doing hex-based serialization in different ways in pepper vs Rust authenticator.

Reply:
fixed